### PR TITLE
Add featured sites to collection page

### LINKED ISSUE #91: SWAP homepage featured sites
Here are the featured sites:

SLAC first web page
https://swap.stanford.edu/was/*/http:/slacvm.slac.stanford.edu/FIND/default.html
https://purl.stanford.edu/qs852wh8529

ShanghaiPRIDE
https://swap.stanford.edu/was/20170323182036/http://www.shpride.com/
https://purl.stanford.edu/dv443xj0944

Bureau of Alcohol, Tobacco, Firearms and Explosives (ATF) Freedom of Information Act (FOIA)
https://swap.stanford.edu/was/20170311062618/https://www.atf.gov/resource-center/freedom-information-act-foia
https://purl.stanford.edu/gf153xn2270

A Vision for Stanford
https://swap.stanford.edu/was/20180518001602/https://ourvision.stanford.edu/
https://purl.stanford.edu/fq241ty7810

Design from Astrid:

Web archive portal home:
https://invis.io/FV12SVON7CSA#/468101257_home_5

Example of gray banner on archived site:
https://invis.io/FV12SVON7CSA#/468101027_banner

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made? 🤔

Fixes #91

This commit also includes:
* Remove hidden, unused divs from DOM
* Fix indentation in collection template

swap_featured_sites

## How was this change tested? 🤨

CI + tested/approved in stage

### CODE REVIEW

File: pywb/templates/search.html [hunk 1]
```diff
@@ -13,178 +13,208 @@
...
+      </div>
+      -->
+    </div>
+    <div class="form-row mt-5 pt-3">
+      <button type="submit" class="btn btn-outline-primary float-left" role="button" aria-label="Submit">
+        {% trans %}Submit{% endtrans %}
+      </button>
+    </div>
+  </form>
+  <div class="row mt-5 featured-sites-header-container">
```
Comment (mjgiarlo) [inline pywb/templates/search.html:152]: Above this point are just indentation/whitespace fixes.

**Comment by peterchanws** [comment]
Below "SLAC first web page":
SLAC Earliest Websites Collection
https://searchworks.stanford.edu/view/wk956yb8565
Please add the following:

Below "ShanghaiPRIDE":
Chinese NGO Web Archive Collection
https://searchworks.stanford.edu/view/jt898xc8096

Below "Bureau of Alcohol, Tobacco, Firearms and Explosives (ATF) Freedom of Information Act (FOIA)":
Freedom of Information Act Collection
https://searchworks.stanford.edu/view/bs665fm2060

Below "A Vision for Stanford":
Stanford University website collection
https://searchworks.stanford.edu/view/9516031

File: pywb/templates/search.html [hunk 2]
```diff
@@ -13,178 +13,229 @@
...
+    <div class="form-row mt-2">
+      <div class="col-5">
+        <div class="custom-control custom-checkbox custom-control">
+          <input type="checkbox" class="custom-control-input" id="open-results-new-window">
+          <label class="custom-control-label" for="open-results-new-window">{{ _('Open results in new window') }}</label>
+        </div>
+      </div>
+    </div>
+    <div class="mt-3">
+      <!-- Disabled at product owner's request
```
Comment (mjgiarlo) [inline pywb/templates/search.html:46]: Comment out hidden div.

File: pywb/templates/search.html [hunk 3]
```diff
@@ -13,178 +13,229 @@
...
+              <div class="input-group-text">{% trans %}To:{% endtrans %}</div>
             </div>
+            <input id="dt-to" type="number" name="date-range-to" class="form-control" pattern="^\d{4,14}$">
+            <div class="invalid-feedback" id="dt-to-bad">
+              {% trans %}Please enter a valid <b>To</b> timestamp. Timestamps may be 4 <= ts <=14 digits{% endtrans %}
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- Disabled at product owner's request
```
Comment (mjgiarlo) [inline pywb/templates/search.html:95]: Comment out hidden div.

**Comment by lwrubel** [comment]
@peterchanws I didn't catch it before, but the thumbnail for A Vision for Stanford is missing the background graphic. Would you want a better thumbnail for the home page (and SearchWorks)?

**Comment by peterchanws** [comment]
@lwrubel Sure. Thanks, Laura, for catching that.

**Comment by lwrubel** [comment]
I'm seeing that we use the thumbnail that's in SearchWorks as the image file, so seeing the updated image would be dependent on the cache eventually showing the new thumbnail. @mjgiarlo while it's nice to point to the stacks image, I'm wondering if we want to just add these to the repo, once @peterchanws has created new thumbnails. Are there reasons to not do this?

**Comment by mjgiarlo** [comment]
@lwrubel 💬 
> I'm seeing that we use the thumbnail that's in SearchWorks as the image file, so seeing the updated image would be dependent on the cache eventually showing the new thumbnail. @mjgiarlo while it's nice to point to the stacks image, I'm wondering if we want to just add these to the repo, once @peterchanws has created new thumbnails. Are there reasons to not do this?

I don't think so? The benefit of using local image assets is better quality control, taking the network out of the picture. The cost is that we need to manually update the was-pywb repo whenever we want new thumbs. I think it's appropriate to shift the costs from our users to us developers. So I'll mark this PR as being held by waiting on the four new thumbnail images.

**Comment by peterchanws** [comment]
Thumbnail regenerated for
https://argo.stanford.edu/view/druid:fq241ty7810
https://argo.stanford.edu/view/druid:dv443xj0944

**Review by lwrubel — APPROVED** [review]

**Pull request merged**